README.md

```
@@ -1,13 +1,14 @@
+of the internal values of the arrays.
 
-```
-forge init -t rainprotocol/foundry.template <projectname>
-cd <projectname>
-forge install foundry-rs/forge-std
-```
-
-Then update the readme, set the docs url and configure github pages on github repo settings.
+Conversions involve mutations to the binary data in the process of moving between
+types. For example, moving from `uint256[]` to `bytes` involves mutating the
+length of the collection even if the binary data values are unchanged.
```

markdownlint-cli2 (0.18.1)

Files should end with a single newline character

(MD047, single-trailing-newline)